utils/fix-readmes.sh

```diff
@@ -34,7 +31,10 @@ function fix_readmes() {
         if [[ -f "${child}/Kptfile" ]]; then
             # kpt package
             echo "Generating ${child}/README.md"
-            utils/kpt-pkg-readme.sh "${child}" > "${child}/README.md"
+            pushd ${child}
+             pkgName=$(basename $child)
+             kpt fn eval -i gcr.io/kpt-fn-contrib/generate-kpt-pkg-docs:unstable --image-pull-policy never --include-meta-resources --mount type=bind,src="$(pwd)",dst=/tmp,rw=true -- readme-path=/tmp/README.md repo-path="https://github.com/GoogleCloudPlatform/blueprints.git/${parent}/" pkg-name="${pkgName}"
```

Comment:
TODO: after https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/722 and release

```suggestion
             kpt fn eval -i gcr.io/kpt-fn-contrib/generate-kpt-pkg-docs:unstable  --include-meta-resources --mount type=bind,src="$(pwd)",dst=/tmp,rw=true -- readme-path=/tmp/README.md repo-path="https://github.com/GoogleCloudPlatform/blueprints.git/${parent}/" pkg-name="${pkgName}"
```